Dockerfile

```diff
@@ -85,13 +116,14 @@ RUN  cd /tmp/api-gateway/ \
     && cd /tmp/api-gateway/api-gateway-hmac-${HMAC_LUA_VERSION} \
     && make install \
          LUA_LIB_DIR=${_exec_prefix}/api-gateway/lualib \
-         INSTALL=/tmp/api-gateway/ngx_openresty-${OPENRESTY_VERSION}/build/install \
+         INSTALL=/tmp/api-gateway/openresty-${OPENRESTY_VERSION}/build/install \
     && echo " ... installing api-gateway-request-validation ..." \
     && tar -xf /tmp/api-gateway/api-gateway-request-validation-${REQUEST_VALIDATION_VERSION}.tar.gz -C /tmp/api-gateway/ \
     && cd /tmp/api-gateway/api-gateway-request-validation-${REQUEST_VALIDATION_VERSION} \
     && make install \
          LUA_LIB_DIR=${_exec_prefix}/api-gateway/lualib \
-         INSTALL=/tmp/api-gateway/ngx_openresty-${OPENRESTY_VERSION}/build/install \
+         INSTALL=/tmp/api-gateway/openresty-${OPENRESTY_VERSION}/build/install \
+    && apk del g++ gcc make \
```

Comment:
@clehene good question, I'm really unclear on this pattern of building and running from the separate or different containers. @amuraru had the same idea, I think, but I don't know.